__tests__/components/navigation.js

```diff
@@ -1,37 +1,74 @@
 import { render, fireEvent, waitFor } from "@testing-library/react";
 import Navigation from "../../app/components/navigation/navigation";
-import { SessionProvider } from "../../app/contexts/session";
+import { SessionContext } from "../../app/contexts/session";
```

Comment:
How come we changed this from `SessionProvider` to `SessionContext` or anywhere, really.

Reply:
Because we need to mock out the setStates that are in the Provider, which we can't do in the actual SessionProvider component